setup.py

```diff
@@ -1,5 +1,14 @@
 from setuptools import setup
 
+EXTRAS_REQUIRE = {
+    "docs": ("sphinx", "pallets-sphinx-themes"),
+    "jwt": ("PyJWT>=1.4.0", "cryptography>=2.0.0"),
+    "tests": ("coverage", "psycopg2-binary", "pytest"),
```

Comment:
why wouldn't tox go in tests tho, since it's used to run tests?

Reply:
That would cause `tox` to install itself: https://github.com/4Catalyzer/flask-resty/blob/86b2048ae4dce487774d5582529a434483baeef0/tox.ini#L10-L11